src/providers/object-provider.js

```diff
@@ -64,6 +69,8 @@ export default class YamcsObjectProvider {
             eventsObject.identifier,
             commandsObject.identifier
         );
+
+        this.openmct.on('destroy', this.#unsubscribeFromAll);
```

Comment:
Does this definitely work? Private functions generally can't be used in callbacks.

Reply:
Yeah, it was getting called when I tested it.